src/renderers/react/WrapperControls.tsx

```diff
@@ -32,15 +32,14 @@ const Button = styled("button")<{ expanded?: boolean }>`
   appearance: none;
   background: ${neutral[93]};
   border: none;
-  border-top: 1px solid ${neutral[100]};
   color: ${neutral[0]};
   cursor: pointer;
   flex-grow: ${({ expanded }) => (expanded ? "1" : "0")};
-  ${({ expanded }) => !expanded && `height: ${buttonWidth}px;`};
   font-size: 15px;
   line-height: 1;
   padding: ${space[1]}px;
   width: ${buttonWidth}px;
+  height: ${buttonWidth}px;
```

Comment:
This might cause problems, I couldn't see what the use cases for 'expanding' were but would be good to test.

Reply:
I think `expanded` gets used to set the height of the buttons depending on the size of the element, but it feels like the `flex-grow` property is maybe doing the job now?

Either way can't see any of the buttons looking out of the ordinary, so seems to be working.